DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java

```diff
@@ -0,0 +1,67 @@
+package net.ravendb.demo.basics.createDocument;
+
+//region Usings
+import net.ravendb.client.documents.session.IDocumentSession;
+//endregion
+import net.ravendb.demo.common.models.Company;
+import net.ravendb.demo.common.models.Contact;
+import net.ravendb.demo.common.DocumentStoreHolder;
+
+public class CreateDocument {
+
+    public void run(RunParams runParams) {
+        String companyName = runParams.getCompanyName();
+        String companyPhone = runParams.getCompanyPhone();
+        String contactName = runParams.getContactName();
+        String contactTitle = runParams.getContactTitle();
+
+        String theNewDocumentId;
+
+        //region Demo
+        //region Step_1
+        Company newCompany = new Company();
+        newCompany.setName(companyName);
+        newCompany.setPhone(companyPhone);
+
+        Contact newContact = new Contact();
+        newContact.setName(contactName);
+        newContact.setTitle(contactTitle);
+        newCompany.setContact(newContact);
+        //endregion
+
+        try (IDocumentSession session = DocumentStoreHolder.store.openSession()) {
+
+            //region Step_2
+            session.store(newCompany);
+            //endregion
+
+            //region Step_3
+            theNewDocumentId = newCompany.getId();
+            //endregion
+
+            //region Step_4
+            session.saveChanges();
+            //endregion
+        }
+        //endregion
+    }
+
+    public static class RunParams {
+        private String companyName;
+        private String companyPhone;
+        private String contactName;
+        private String contactTitle;
+
+        public String getCompanyName() { return companyName; }
```

Comment:
did you auto generate this using Rider?

Reply:
Actually - no - I just typed :)
I know InteliJ has the 'Create getter and setter'
It seems that Rider doesn't have this option

Reply:
oh I meant IntellijJ.

Please remove this code and regenerate to having standard formatting. 

In Rider there is no need to generate. C# syntax is more concise.

Reply:
ok - pushed fix